docs/docs/genai/version-tracking/track-application-versions-with-mlflow.mdx

```diff
@@ -70,7 +82,7 @@ MLflow automatically captures Git metadata whenever you log models or start runs
 Install MLflow and required packages:
 
 ```bash
-pip install --upgrade "mlflow>=3.1" openai
+pip install "mlflow>=3.4" openai
```

Comment:
Can we clearly state somewhere (e.g. in a ::note block) that MLflow >= 3.4 is required?